test/functional/p2p-fingerprint.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python3
+# Copyright (c) 2017 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test various fingerprinting protections.
+
+If an orphan block more than a month old or its header are requested by a peer,
+the node should pretend that it does not have it to avoid fingerprinting.
+"""
+
+import time
```

Comment:
Nit
```py
from time import (sleep, time)
```

Reply:
If other people feel strongly, I'm willing to change it, but I prefer to [only import packages and modules](https://google.github.io/styleguide/pyguide.html?showone=Imports#Imports), not values. This makes it a bit more clear where the function is coming from.